src/devtools/views/Components/SelectedElement.js

```diff
@@ -267,80 +265,3 @@ function OwnerView({ displayName, id }: { displayName: string, id: number }) {
     </button>
   );
 }
-
-function hydrateHelper(dehydratedData: DehydratedData | null): Object | null {
-  if (dehydratedData !== null) {
-    return hydrate(dehydratedData.data, dehydratedData.cleaned);
-  } else {
-    return null;
-  }
-}
-
-function useInspectedElement(id: number | null): InspectedElement | null {
-  const idRef = useRef(id);
-  const bridge = useContext(BridgeContext);
-  const store = useContext(StoreContext);
-
-  const [inspectedElement, setInspectedElement] = useState(null);
-
-  useEffect(() => {
-    // Track the current selected element ID.
-    // We ignore any backend updates about previously selected elements.
-    idRef.current = id;
-
-    // Hide previous/stale insepected element to avoid temporarily showing the wrong values.
-    setInspectedElement(null);
-
-    // A null id indicates that there's nothing currently selected in the tree.
-    if (id === null) {
-      return () => {};
-    }
-
-    const rendererID = store.getRendererIDForElement(id);
-
-    // Update the $r variable.
-    bridge.send('selectElement', { id, rendererID });
```

Comment:
Was this intentionally removed? Seems like it wouldn't update `$r` now.

Reply:
(If you intentionally removed it, probably should remove the handler in agent too.)

Reply:
I didn't intentionally remove the $r update. I did intentionally remove the imperative hook. Thanks for pointing out the $r bit.